README.md

```diff
@@ -747,6 +747,44 @@ The default reader `spacy.FewShotReader.v1` supports `.yml`, `.yaml`, `.json` an
 path = "lemma_examples.yml"
 ```
 
+#### spacy.Sentiment.v1
+
+Performs sentiment analysis on provided texts. Scores between 0 and 1 are stored in `Doc._.sentiment` - the higher, the
+more positive. Note in cases of parsing issues (e. g. in case of unexpected LLM responses) the value might be `None0-`
```

Comment:
Can't we transform this to `None` or `0` when that happens? 🤔

Reply:
Ah, the value is `None`. That's a typo. Fixed in https://github.com/explosion/spacy-llm/pull/200/commits/02d90478cd8b8ea1354b8eba7fa0d7d8f93331b3.

Reply:
Haha, ok, that makes a lot more sense!